legate/driver/command.py

```diff
@@ -24,7 +24,7 @@
     from .config import ConfigProtocol
     from .launcher import Launcher
 
-__all__ = ("CMD_PARTS",)
+__all__ = ("_CMD_PARTS_SHARED", "CMD_PARTS_LEGION", "CMD_PARTS_CANONICAL")
```

Comment:
No need to "export" private names
```suggestion
__all__ = ("CMD_PARTS_LEGION", "CMD_PARTS_CANONICAL")
```